docs/docsite/rst/porting_guide_2.5.rst

```diff
@@ -103,6 +103,32 @@ See :doc:`playbooks_tests` for more information.
 
 Additionally, a script was created to assist in the conversion for tests using filter syntax to proper jinja test syntax. This script has been used to convert all of the Ansible integration tests to the correct format. There are a few limitations documented, and all changes made by this script should be evaluated for correctness before executing the modified playbooks. The script can be found at `https://github.com/ansible/ansible/blob/devel/hacking/fix_test_syntax.py <https://github.com/ansible/ansible/blob/devel/hacking/fix_test_syntax.py>`_.
 
+Using a loop on a package module via squash_actions
+---------------------------------------------------
+
+The use of ``squash_actions`` to invoke a package module, such as "yum", to only invoke the module once is deprecated, and will be removed in Ansible 2.9.
+
+Instead of relying on implicit squashing, tasks should instead supply the list directly to the ``name``, ``pkg`` or ``package`` parameter of the module. This functionality has been supported in most modules since Ansible 2.3.
```

Comment:
How does someone know if the module supports the new way?

Reply:
There isn't really any easy way for a person to determine it, unless they either:

1. try
2. read the source of the module

If the person gets the deprecation message, then they are using one of those modules that supports it.

At the time of writing this, the following modules support this:

```
apk.py:            name=dict(type='list'),
apt.py:            package=dict(type='list', aliases=['pkg', 'name']),
dnf.py:            name=dict(aliases=['pkg'], type='list'),
homebrew_tap.py:            name=dict(aliases=['tap'], type='list', required=True),
openbsd_pkg.py:            name=dict(type='list', required=True),
pacman.py:            name=dict(type='list', aliases=['package', 'pkg']),
pkg5.py:            name=dict(type='list', required=True),
pkgin.py:            name=dict(aliases=["pkg"], type='list'),
pkgng.py:            name=dict(aliases=["pkg"], required=True, type='list'),
portage.py:            package=dict(default=None, aliases=['name'], type='list'),
rhsm_repository.py:            name=dict(type='list', required=True),
slackpkg.py:            name=dict(aliases=["pkg"], required=True, type='list'),
sorcery.py:            name=dict(default=None, aliases=['spell'], type='list'),
xbps.py:            name=dict(default=None, aliases=['pkg', 'package'], type='list'),
yum.py:            name=dict(type='list', aliases=['pkg']),
yum_repository.py:        includepkgs=dict(type='list'),
zypper.py:            name=dict(required=True, aliases=['pkg'], type='list'),
```